localisation/000_Other_title_names.csv

```diff
@@ -293,9 +293,9 @@ d_shamanism;The Shamanic Council;Le Conseil chamanique;Der schamanische Rat;;El
 d_shamanism_adj;Shamanic;Chamanique;schamanisch;;Cham�nico;;;;;;;;;x
 d_earth_mother_worship;Earth-Temple;Temple de la Terre;Tempel der Erde;;Templo de la Tierra;;;;;;;;;x
 d_earth_mother_worship_adj;Earthen;En Terre;irdisch;;De barro;;;;;;;;;x
-d_tidemother;The Tidesages;Des Eaugures;Die Gezeitenweisen;;Los Sabiomar;;;;;;;;;x
+d_tidemother;Tidesages;Eaugures;Gezeitenweisen;;Los Sabiomar;;;;;;;;;x
```

Comment:
Are titles adjusted in landed_titles to be marked as having "the" in front now that it is removed from title loc? e.g. "the Byzantine Empire" instead of "Byzantine Empire" in game.

Reply:
Removed it, because it's religious heads title. When you abandon the religion, it says something like "The The Tidesages..."

Reply:
I know, I'm just referring to that little checkbox that one selects when customizing their titles in game where it adds "the" when listing the characters full titled name. It doesn't affect event text.